### ISSUE TITLE
What is @bbb-fe ???

### ISSUE THREAD

**Issue by cod3r0k** [bigbluebutton/bbb-install#681]
I installed the latest 2.7 version via the below code:

``` wget -qO- https://raw.githubusercontent.com/bigbluebutton/bbb-install/v2.7.x-release/bbb-install.sh | bash -s -- -w -v focal-270 -s .... ```

after that, my nginx error is:

 could not find named location "@bbb-fe"

**Comment by cod3r0k** [comment]
Is @bbb-fe related for some debugs or .... ? @ffdixon

**Comment by antobinary** [comment]
Hi @cod3r0k 
Could you please copy-paste some more context? This `@bbb-fe` does not ring a bell yet.

**Comment by AdamZWinter** [comment]
Just happed to be running into the problem and found this here.   @bbb-fe is in the nginx config flile:

  # BigBlueButton landing page.
  location / {
    root   /var/www/bigbluebutton-default/assets;
    try_files $uri @bbb-fe;
  }

I installed Greenlight along with BBB (on the same server with the -g flag), and I see the docker container running with 127.0.0.1 ->3000 (which I assume is a React frontend), but I don't see anything in the nginx configuration, nor the haproxy configuration that connects to it.

**Comment by ffdixon** [comment]
See

 https://github.com/bigbluebutton/bbb-install/blob/v2.7.x-release/bbb-install.sh#L1641

**Comment by Xsmael** [comment]
Have you guys solved this ? i have the same issue!

**Comment by AdamZWinter** [comment]
Yes, check out this thread:  https://github.com/bigbluebutton/bbb-install/issues/700

**Comment by emostoyanov** [comment]
> Have you guys solved this ? i have the same issue!

Hi Xsmael,

I faced the same issue - apparently `@bbb-fe` is supposed to be replaced with the Greenlight v3 url (RELATIVE_URL_ROOT in .env, e.g. /gl). This did not happened in `/etc/nginx/sites-available/bigbluebutton` file at the install time and a possible workarround is to replace the line

    `try_files $uri @bbb-fe;`
    
 with 
 
     `return 301 /gl;`

I also tried first to change the line to `try_files $uri /gl;`, but this loads empty page - gl interface does not show up, although viewing the page source it should.

**Comment by AdamZWinter** [comment]
NO.  @bbe-fe is NOT a template placeholder that was supposed to be replaced.  It is fine like it is.  Your problem is not there.  See this thread:  /usr/share/bigbluebutton/nginx/greenlight-v3.nginx

and then (if don't already know how) learn how to do everything in this thread:  https://github.com/bigbluebutton/bbb-install/issues/700
